pkg/image/controller/signature/signature_import_controller.go

```diff
@@ -107,7 +107,9 @@ func (s *SignatureImportController) work() bool {
 	err := s.syncImageSignatures(key.(string))
 	if err != nil {
 		utilruntime.HandleError(fmt.Errorf("error syncing image %s, it will be retried: %v", key.(string), err))
-		s.queue.AddRateLimited(key)
+		if s.queue.NumRequeues(key) < 5 {
```

Comment:
Pull the magic number out

Reply:
pull it out to where?  my goal here was to say "hey, if we can't retrieve this after a couple tries, it's not important enough to keep retrying" (but we'll eventually retry on relist of course).